### PR TITLE
Update RuleSets

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: src/RuleSet/Sets/PHP80MigrationSet.php [hunk 1]
```diff
@@ -23,6 +23,7 @@ public function getRules()
     {
         return [
             '@PHP73Migration' => true,
+            'no_unneeded_final_method' => true,
```
Comment (keradus) [inline src/RuleSet/Sets/PHP80MigrationSet.php:26]: can you put a comment why it's matching PHP 8.0 migration?

File: src/RuleSet/Sets/SymfonySet.php [hunk 2]
```diff
@@ -179,6 +179,7 @@ public function getRules()
             ],
             'single_line_throw' => true,
             'single_quote' => true,
+            'single_space_after_construct' => true,
```
Comment (keradus) [inline src/RuleSet/Sets/SymfonySet.php:182]: nice!

File: src/RuleSet/Sets/PHP80MigrationSet.php (hunk 1 above)
Comment (SpacePossum) [reply src/RuleSet/Sets/PHP80MigrationSet.php:26]: sure, it is here https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5219 , I'll add it to the code

**Review by keradus — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**